packages/core-mobile/app/new/features/browser/components/BrowserItem.tsx

```diff
@@ -51,7 +53,34 @@ export const GridItem = memo(
           },
           style
         ]}>
-        <Avatar image={image} size={48} />
+        <View>
+          {isFavorite ? (
+            <BlurView
```

Comment:
let's not use BlurView on Android. also, is it possible to use the BlurViewWithFallback component?